tests/core/test_sync_monitor_stateful.py

```diff
@@ -48,6 +48,8 @@ def recursive_compare_fs_dumps(alice_dump, bob_dump, ignore_need_sync=False):
 
 
 @pytest.mark.slow
+# This test runs into infinite loop when it uses real data storage
```

Comment:
Are you sure this is an infinite loop ? hypothesis tests are slow, and can become very very slow if the code behind it is itself slow ^^